src/client/Microsoft.Identity.Client/PublicApi/net462/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,3 @@
 Microsoft.Identity.Client.AssertionRequestOptions.ClientCapabilities.get -> System.Collections.Generic.IEnumerable<string>
-Microsoft.Identity.Client.AssertionRequestOptions.ClientCapabilities.set -> void
\ No newline at end of file
+Microsoft.Identity.Client.AssertionRequestOptions.ClientCapabilities.set -> void
```

Comment:
merging conflict?

Reply:
I think this is just a github artifact. No changes were made to this line